src/components/Settings/integrations/integrationsHelper.ts

```diff
@@ -15,7 +16,7 @@ export const StyledServicesButton = styled(Button)(({ theme }) => ({
 })) as typeof Button; // Type cast so that `component` prop works
 
 export interface AccordionProps {
-  handleAccordionChange: (panel: string) => void;
-  expandedPanel: string;
+  handleAccordionChange: (accordion: IntegrationAccordion | null) => void;
+  expandedAccordion: IntegrationAccordion | null;
```

Comment:
I wonder if you could expand on the `AccordionProps` interface in `accordionHelper.